docs/docs/guides/database/05-migrations.md

```diff
@@ -73,6 +73,12 @@ export default class Users extends BaseSchema {
 }
 ```
 
+Next, you will need run the following ace command to build the application:
+
+```sh
+node ace build
+```
```

Comment:
We should add this as a note.

[note]

Migrations are executed against the compiled Javascript code. So, make sure to run one of the following ace command to keep the compiled output upto date.

```sh
# watch & re-compile
node ace build --watch

# watch, re-compile & start http server
node ace serve --watch
```

[/note]

Reply:
@thetutlage I've updated as per your comment